scripts/dnscheck.py

```diff
@@ -0,0 +1,40 @@
+import dns.resolver
+import sys
+
+def check_domain(domain):
+    try:
+        # Query for 'A' records
+        dns.resolver.resolve(domain, "A")
+        has_a_record = True
+    except (dns.resolver.NXDOMAIN, Exception):
+        has_a_record = False
+    
+    try:
+        # Query for 'NS' records
+        dns.resolver.resolve(domain, "NS")
+        has_ns_record = True
+    except (dns.resolver.NXDOMAIN, Exception):
+        has_ns_record = False
```

Comment:
maybe do it like this? so that we only remove a domain if we are sure that it does not exist anymore? otherwise not.

Something like this:
```py3
try: 
  dns.resolver.resolve(domain, "NS")
  has_ns_record = True
except dns.resolver.NXDOMAIN:
  has_ns_record = False
except:
  has_ns_record = True
```

Reply:
Done, can you check the latest changes? Domains now get automatically removed as well!